src/main/resources/hudson/plugins/git/GitSCM/help-useExistingAccountWithSameEmail.html

```diff
@@ -0,0 +1,6 @@
+<p>
+    Will make sure that user will be searched by their actual e-mail address before resolving to creating the user with its mail address as id.<br/>
```

Comment:
I think you mean:

> searched by their actual e-mail address before **resorting** to creating the user

rather than 

> searched by their actual e-mail address before **resolving** to creating the user

Reply:
corrected